docs/Feature Documentation/Multiple Transports/index.md

```diff
@@ -0,0 +1,82 @@
+# General Description
+
+The Multiple Transports feature allows apps connected to Core to start another connection over a different transport for certain services (for example, an app connected over bluetooth using WiFi as a secondary transport for video streaming). This guide will give an overview of the process which is used to establish a secondary transport connection. See [SDL-0141](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0141-multiple-transports.md) for more details on the original feature proposal.
```

Comment:
```suggestion
The Multiple Transports feature allows apps connected to SDL Core to start another connection over a different transport for certain services. For example, an app connected over bluetooth may use WiFi as a secondary transport for video streaming. This guide will give an overview of the process which is used to establish a secondary transport connection. See [SDL-0141](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0141-multiple-transports.md) for more details on the original feature proposal.
```

Reply:
I think "Bluetooth" and "Secondary Transport" should be capitalized.  

I'd also change the proposal reference to "See [SDL 0141 - Supporting Simultaneous Multiple Transports](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0141-multiple-transports.md) for more..."